hack/generate/samples/internal/go/memcached-with-webhooks/e2e_test_code.go

```diff
@@ -288,15 +288,13 @@ var _ = Describe("memcached", func() {
 
 			By("validating that pod(s) status.phase=Running")
 			getMemcachedPodStatus := func() error {
-				cmd = exec.Command("kubectl", "get",
-					"pods", "-l", "app=memcached", "-o", "jsonpath={.items[*].status}",
-					"-n", namespace,
+				status, err := kbc.Kubectl.Get(true, "pods", "-l",
```

Comment:
@NikhilSharmaWe this one is an e2e test scaffold on the sample .
we do have not here kbc command. We need to do the same check but change the label used to get the memcahed deployment. , see that before we were running the kubectl command.

Reply:
yes I am making changes

Reply:
See that the only change that we need from the original code to the new one is replace ` "app=memcached"` with `"app.kubernetes.io/name=memcached"`